install/operator/pkg/syndesis/capabilities/capabilities.go

```diff
@@ -19,13 +19,16 @@ package capabilities
 import (
 	"errors"
 	"fmt"
+	"strconv"
+	"strings"
 
 	errs "github.com/pkg/errors"
 	"github.com/syndesisio/syndesis/install/operator/pkg/syndesis/clienttools"
 )
 
 type ApiServerSpec struct {
 	Version          string // Set to the kubernetes version of the API Server
+	Openshift4       bool   // set to true if running on openshift 4
```

Comment:
Not sure I like this upstream.

Reply:
This is to make a distinction of openshift 3 or 4. I could use the `consolelinks` as condition, but using the `Openshift4` field is a clear drive to the right openshift management console 4 in the `configuration.SetOpenshiftManagementConsoleUrl` function.